plugins/basestate/include/yarp/dev/BaseStateDriver.h

```diff
@@ -49,9 +49,10 @@ namespace yarp
 /**
  * YARP device driver to give the gazebo state information of a desired base link of the robot
  * The state information include 
- * - absolute pose of the base link (x, y, z. roll, pitch, yaw)
+ * - absolute pose of the base link (x, y, z. roll, pitch, yaw) 
```

Comment:
```suggestion
 * - absolute pose of the base link (x, y, z. roll, pitch, yaw)
```
Please don't add trailing whitespace.

Reply:
Sorry about that. Fixed in https://github.com/robotology/gazebo-yarp-plugins/commit/0c9c3cde184bcf9192e51d61ddc96da404ace358